rkt/cli_apps.go

```diff
@@ -440,6 +441,43 @@ func (au *appSeccompFilter) Type() string {
 	return "appSeccompFilter"
 }
 
+// appOOMScoreAdj is to adjust /proc/$pid/oom_score_adj
+type appOOMScoreAdj apps.Apps
+
+func (aml *appOOMScoreAdj) Set(s string) error {
+	app := (*apps.Apps)(aml).Last()
+	if app == nil {
+		return fmt.Errorf("--oom-score-adj must follow an image")
+	}
+	limit, err := strconv.Atoi(s)
+	if err != nil {
+		return err
+	}
+	isolator := types.LinuxOOMScoreAdj(limit)
+	if err := isolator.AssertValid(); err != nil {
```

Comment:
nit: Can have a constructor e.g. `types.NewLinuxOOMScoreAdj()` that does assertvalid() inside.